pkg/operator/hive/clustersync.go

```diff
@@ -0,0 +1,137 @@
+package hive
+
+import (
+	"strconv"
+
+	log "github.com/sirupsen/logrus"
+
+	corev1 "k8s.io/api/core/v1"
+
+	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
+	hiveconstants "github.com/openshift/hive/pkg/constants"
+	"github.com/openshift/hive/pkg/controller/images"
+	controllerutils "github.com/openshift/hive/pkg/controller/utils"
+	"github.com/openshift/hive/pkg/operator/assets"
+	"github.com/openshift/hive/pkg/operator/util"
+	"github.com/openshift/hive/pkg/resource"
+)
+
+func (r *ReconcileHiveConfig) deployClusterSync(hLog log.FieldLogger, h resource.Helper, instance *hivev1.HiveConfig) error {
+	asset := assets.MustAsset("config/clustersync/statefulset.yaml")
```

Comment:
You're well positioned here, lets just lump in that other card we talked about, add a setting in HiveConfig for configuring the number of replicas. https://issues.redhat.com/browse/CO-1290 can be deleted once done.

Reply:
Just to be clear, you're asking for that card to be done as part of this PR, correct?

Reply:
Yes please.